packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -77,10 +79,12 @@ class AppSizeController {
 
   int nodeIndexCalculator(TreemapNode newAnalysisRoot) {
     final searchCondition = (TreemapNode n) => n == newAnalysisRoot;
-    return newAnalysisRoot.root.childCountToMatchingNode(
+    if (!newAnalysisRoot.root.isExpanded) newAnalysisRoot.root.expand();
+    final nodeIndex = newAnalysisRoot.root.childCountToMatchingNode(
       matchingNodeCondition: searchCondition,
       includeCollapsedNodes: false,
     );
+    return isDeferredApp ? nodeIndex - 1 : nodeIndex;
```

Comment:
Let's also check if `deferredLoadingSupportEnabled` is true

Reply:
Should this check only be placed here if it's not present in all the other deferred loading support changes?